docs/tilequery.md

```diff
@@ -0,0 +1,50 @@
+# Tilequery
+
+The `Tilequery` class provides access to the Mapbox Tilequery API.  You can import it from either the `mapbox` module or the `mapbox.services.tilequery` module.
+
+__mapbox__:
+
+```python
+>>> from mapbox import Tilequery
+
+```
+
+__mapbox.services.tilequery__:
+
+```python
+>>> from mapbox.services.tilequery import Tilequery
+
+```
+
+See https://www.mapbox.com/api-documentation/#tilequery for general documentation of the API.
+
+Use of the Tilequery API requires an access token, which you should set in your environment.  For more information, see the [access tokens](access_tokens.md) documentation.
+
+## Tilequery Method
+
+The public method of the `Tilequery` class provides access to the Tilequery API and returns an instance of [`requests.Response`](http://docs.python-requests.org/en/latest/api/#requests.Response).
+
+
+## Usage: Retrieving Features
+
+Instantiate `Tilequery`.
+
+```python
+>>> tilequery = Tilequery()
+
+```
+
+Call the `tilequery` method, passing in values for `map_id`, `lon`, and `lat`.  Pass in values for optional arguments as necessary - `radius`, `limit`, `dedupe`, `geometry`, and `layers`.
+
+```python
+>>> response = tilequery.tilequery("mapbox.mapbox-streets-v10", 0.0, 1.1)
```

Comment:
@sgillies If I'm not mistaken, the highest version of the `mapbox.mapbox-streets-v*` tileset is `v8` - does that sound right to you?

Reply:
@HeyStenson the python module author chose "v10" for some other reason. Putting a valid version in there is definitely an improvement!

Reply:
This was a mistake on my part.  In the `Styles` section of the API docs, there is a reference to `mapbox://styles/mapbox/streets-v10`.  I may have had this in mind when working on the tests and docs for this module.

Please let me know if I should submit a pull request with `v8` instead of `v10`.  Thank you!

Reply:
@critical-path we would be grateful, but it's not critical to fix this.

Reply:
Hi @critical-path! 👋 Yes, it'd be awesome if you could open a PR with `v8`. Thank you!

I totally understand the confusion! The `v10` reference you saw in the Styles API documentation refers to v10 of the [Mapbox Streets _style_](https://www.mapbox.com/maps/streets/). This is a different beast than the [Mapbox Streets _vector tileset_](https://www.mapbox.com/vector-tiles/mapbox-streets-v8/), which is on v8 and is what this example would query. A [map style](https://www.mapbox.com/help/define-style/) dictates how a map will be rendered, while a [tileset](https://www.mapbox.com/help/define-tileset/) determines what data will be used in the map.

Reply:
Thanks for the clarification @HeyStenson !